src/protocols/PhysicalLayer.h

```diff
@@ -77,11 +77,11 @@ class PhysicalLayer {
 
       \param str Address of Arduino String to save the received data.
 
-      \param len Expected number of characters in the message.
+      \param len Expected number of characters in the message. Leave as 0 if expecting a unknown size packet
 
       \returns \ref status_codes
     */
-    int16_t receive(String& str, size_t len = 0);
+   int16_t receive(String& str, size_t len = 0) ;
```

Comment:
```suggestion
    int16_t receive(String& str, size_t len = 0);
```

Reply:
Kwrite shows the spacing suggested. If you still find something wrong with the spacing, feel free to change it. :)